control-plane-operator/controllers/hostedcontrolplane/kas/ibmcloud_kms.go

```diff
@@ -155,7 +155,7 @@ func buildVolumeIBMCloudKMSKP(v *corev1.Volume) {
 func buildKASContainerIBMCloudKMS(image string, region string, kmsInfo string, customerAPIKeyReference *corev1.EnvVarSource) func(c *corev1.Container) {
 	return func(c *corev1.Container) {
 		c.Image = image
-		c.ImagePullPolicy = corev1.PullAlways
+		c.ImagePullPolicy = corev1.PullIfNotPresent
```

Comment:
@relyt0925 @hasueki was it deliberate that this is set to `always` for the IBM cloud components? If the registry is down for whatever reason and a pod crashes, it won't be able to start again because it can not reach the registry so IMHO imagePullPolicy Always can become an operational issue

Reply:
No, I don't think there's any reason to set it to `Always`. Right, the reason to change it to `IfNotPresent` makes sense to me. Similar to this issue, https://github.com/openshift/hypershift/issues/1182.